specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualWan.json

```diff
@@ -810,6 +810,248 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/vpnServerConfigurations/{vpnServerConfigurationName}": {
```

Comment:
What happened to the PATCH api? That is needed, at least for `tags` but ideally for all mutable properties

Reply:
We don't need PATCH api the scenario to update tags is not valid for this resource.  Also, we do update mutable properties using PUT api only, this is similar to other resource types's apis design.

Reply:
Added PATCH api back and build is passing now. Thanks!